compiler/optimizer/LoopCanonicalizer.cpp

```diff
@@ -2027,9 +2027,40 @@ void TR_LoopCanonicalizer::eliminateRedundantInductionVariablesFromLoop(TR_Regio
                //   _primaryInductionIncrementBlock = loopTestBlock;
                //   _derivedInductionIncrementBlock = entryBlock;
                //     }
+               static bool disablePostDominatorsCheck = (NULL != feGetEnv("TR_DisablePostDominatorsCheck"));
```

Comment:
could we make this more pass specific something like TR_DisableIVEPostDominatorsCheck - just other things can have post dominance checks.